doc/crdt.md

```diff
@@ -56,9 +48,9 @@ The CRDT state is:
 
   - An undo group id.
 
-  - A set of delete intervals.
```

Comment:
I added that text and a little more further up in the `Subset` section when I first mention characters.